SendToLogit/README.md

```diff
@@ -0,0 +1,35 @@
+# Send a Log entry to Logit.io
+
+### Purpose
+There is a requirement to push certain events into logit.io (ELK) so that it can be presented on a dashboard in Grafana. 
+This routine has been made generic, so that it can be reused if necessary and additional data points can be simply added.
+
+### Input Parameters
+* LOGIT-API-KEY - API Key for Logit
+* JSON          - JSON Payload
+
+
+### Example
+```       
+
+name:  Test
+on:
+  workflow_dispatch:
+
+jobs:
+  LOGIT:
+    runs-on: ubuntu-latest
+    steps:
+    
+       - name: Tests
+         uses: DFE-Digital/github-actions/SendToLogit@logit
```

Comment:
@master ?

Reply:
yes, I cut it from my test, ill change it